src/python/pants/backend/jvm/register.py

```diff
@@ -217,7 +217,7 @@ def register_goals():
 
   # Formatting.
   task(name='scalafmt', action=ScalaFmtFormat, serialize=False).install('fmt')
-  task(name='scalafix', action=ScalaFixFix).install('fmt')
+  task(name='scalafix', action=ScalaFixFix).install(goal='fmt', before='scalafmt')
```

Comment:
Could also just reorder these lines. But this is fine too!

Reply:
oh duh! haha how did i miss that? I will just reverse them. 

If there is other issue to cherry pick, then might as well. Otherwise waiting for 1.9.x is good too.

Reply:
Picked it.